### PR TITLE
Clean up timeline code

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/mfuxsjxj

Depends on: https://github.com/alphagov/collections-publisher/pull/1405

# What's changed?

Remove code added to allow developers to switch between the old and new versions of the timeline on the coronavirus landing page.

# Why?
Once the feature has gone live, this code will no longer be needed.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/coronavirus_landing_page/components/shared/_timeline.html.erb [hunk 1]
```diff
@@ -5,72 +5,54 @@
...
+  <%= form_tag coronavirus_landing_page_path,
+    method: :get,
+    class: "js-change-location" do |f| %>
 
-      <%= render "govuk_publishing_components/components/radio", {
-        name: "nation",
-        heading: t("coronavirus_landing_page.show.timeline.radio.heading"),
-        margin_bottom: 7,
-        items: details.timeline_nations_items
+    <%= render "govuk_publishing_components/components/radio", {
```
Comment (Rosa-Fox) [inline app/views/coronavirus_landing_page/components/shared/_timeline.html.erb:12]: I think this code might be missing, I haven't run it on integration so not 100% sure but I think the title and nesting the details component are still needed?

```
<p class="govuk-body">
        <%=t("coronavirus_landing_page.show.timeline.controls.status_html", nation: details.selected_nation.titleize) %>
      </p>

      <%= render "govuk_publishing_components/components/details", {
        title: t("coronavirus_landing_page.show.timeline.controls.change"),
        data_attributes: {
          track_category: "pageElementInteraction",
          track_action: "TimelineNation",
          module: "govuk-details",
        }
      } do %>
```

File: spec/fixtures/content_store/coronavirus_landing_page.json [hunk 2]
```diff
@@ -392,19 +392,33 @@
...
+          "national_applicability": ["scotland"],
           "paragraph": "People must not meet in groups larger than 6 in England. There are [exceptions to this 'rule of 6'](/government/publications/coronavirus-covid-19-meeting-with-others-safely-social-distancing/coronavirus-covid-19-meeting-with-others-safely-social-distancing#seeing-friends-and-family)\n"
         },
         {
           "heading": "24 July",
+          "national_applicability": ["wales"],
           "paragraph": "[Face coverings are mandatory in shops](/government/publications/face-coverings-when-to-wear-one-and-how-to-make-your-own/face-coverings-when-to-wear-one-and-how-to-make-your-own)\n"
+        },
+        {
+          "heading": "In Northern Ireland",
```
Comment (Rosa-Fox) [inline spec/fixtures/content_store/coronavirus_landing_page.json:414]: It doesn't overly matter but might be good to change these heading values to be dates to match the ones before?

**Review by Rosa-Fox — COMMENTED** [review]
Nice one @leenagupte! I had a couple of small comments. I spent quite a while comparing this against your [PR for displaying by timeline](https://github.com/alphagov/collections/pull/2456/files) and searched through the codebase for method calls in other places etc and this was all that I could find.

I have not run this locally... I am not sure if I can but I am a bit behind as to where we are with deploying this. I will find out tomorrow to see if I can run it!

File: app/views/coronavirus_landing_page/components/shared/_timeline.html.erb (hunk 1 above)
Comment (leenagupte) [reply app/views/coronavirus_landing_page/components/shared/_timeline.html.erb:12]: Ah, the details element was removed in a later PR: https://github.com/alphagov/collections/pull/2465

File: spec/fixtures/content_store/coronavirus_landing_page.json (hunk 2 above)
Comment (leenagupte) [reply spec/fixtures/content_store/coronavirus_landing_page.json]: Good point. Have updated and fixed up.

**Comment by leenagupte** [comment]
@Rosa-Fox Thanks for your review. I've addressed your comments and fixed up.

File: app/views/coronavirus_landing_page/components/shared/_timeline.html.erb (hunk 1 above)
Comment (Rosa-Fox) [reply app/views/coronavirus_landing_page/components/shared/_timeline.html.erb:12]: 👍

File: spec/fixtures/content_store/coronavirus_landing_page.json (hunk 2 above)
Comment (Rosa-Fox) [reply spec/fixtures/content_store/coronavirus_landing_page.json]: 👍

**Review by Rosa-Fox — APPROVED** [review]

**Pull request merged**